docs/src/token-2022/status.md

```diff
@@ -2,17 +2,18 @@
 title: Project Status
 ---
 
-The Token-2022 program is still under audit and not meant for full production use.
-In the meantime, all clusters have the latest program deployed **for testing and
-development purposes ONLY**.
+The Token-2022 program is ready for full production use. All clusters have the
+latest program deployed **without confidential transfer functionality**.
+
```

Comment:
this seems more like a temporary "announcement" than something that needs to live in the docs?

Reply:
How about I get rid of the first sentence? At least knowing what's deployed should be pretty useful